tsconfig.json

```diff
@@ -19,4 +19,5 @@
     "baseUrl": "src",
   },
   "include": ["src"],
+  "exclude": ["*.test.ts"],
```

Comment:
why do we want to exclude tests from typescript compilation?

Reply:
I was trying to exclude from eslint temporarily. Wrong place, I realized but did not revert.